CMakeLists.txt

```diff
@@ -29,8 +29,11 @@ SET(PROJECT_TARGET_BIN "mmex")
 # ./resources/mmex.rc
 # ---------------------------------------------------------------------------
 SET(MoneyManagerEx_VERSION_MAJOR 1)
-SET(MoneyManagerEx_VERSION_MINOR 3)
-SET(MoneyManagerEx_VERSION_PATCH 3)
+SET(MoneyManagerEx_VERSION_MINOR 4)
+SET(MoneyManagerEx_VERSION_PATCH 0)
+SET(MoneyManagerEx_VERSION_ALPHA 0)
```

Comment:
Ok, to the future I prepare another PR which takes into the consideration also test names in artifact naming convention.